.github/workflows/build-push-images.yaml

```diff
@@ -137,6 +137,31 @@ jobs:
         if: steps.filter.outputs.kcp-registrar == 'true'
         run: echo "Image pushed to ${{ steps.push-to-quay-kcp-registrar.outputs.registry-paths }}"
 
+      # Build and push devenv image, tagged with the branch name and the commit SHA.
+      - name: Build devenv Image
+        id: build-image-devenv
+        if: steps.filter.outputs.devenv == 'true'
```

Comment:
In order for this to work, you need to setup the path for this image [here](https://github.com/openshift-pipelines/pipeline-service/blob/main/.github/workflows/build-push-images.yaml#L24).

Reply:
Fixed.